src/routes/Login.tsx

```diff
@@ -36,13 +36,22 @@ export const LoginRoute: React.FC = () => {
         GitHub Notifications <br /> on your menu bar.
       </div>
 
-      <button
+      {/* <button
```

Comment:
It's probably okay as-is, since we plan to get to this soon, but usually I prefer to add a comment explaining why a section is commented out.

For example, if we forget to remove a commented-out section, how will we understand why it's there (and that it can be removed) in six months?

Reply:
Good callout.  I've added a `FIXME` comment and reference the currently known open GH Issues